diff -ur ruby-activerdf_redland-1.2.2-o/lib/activerdf_redland/redland.rb ruby-activerdf_redland-1.2.2/lib/activerdf_redland/redland.rb
--- ruby-activerdf_redland-1.2.2-o/lib/activerdf_redland/redland.rb	1969-12-31 17:00:00.000000000 -0700
+++ ruby-activerdf_redland-1.2.2/lib/activerdf_redland/redland.rb	2009-08-16 12:20:21.474689313 -0600
@@ -2,8 +2,8 @@
 # Copyright:: (c) 2005-2006 Eyal Oren
 # License:: LGPL
 require 'active_rdf'
-require 'federation/connection_pool'
-require 'queryengine/query2sparql'
+require 'active_rdf/federation/connection_pool'
+require 'active_rdf/queryengine/query2sparql'
 require 'rdf/redland'
 
 # Adapter to Redland database
